_posts/2019-01-14-lecture-01.md

```diff
@@ -0,0 +1,310 @@
+Do we use maximum-likelihood estimation, or can we use domain knowledge?</li>
+
+  <li>How can we compute conditional distributions of unobserved (latent) variable without needing to sum over a large number of state configurations?</li>
+</ol>
+
+In the next section, we give an example to show how graphical models provide an effective way of answering these questions. 
+
+***
+
+## Structure within a Cell: real world example of graphical models with structure among the RVs
+
+<figure id="cell-ex-figure" class="l-body-outset">
+  <div class="row">
+    <img src="{{ '/assets/img/notes/lecture-01/cell-ex.png' | relative_url }}" />
+  </div>
+  <figcaption>
+    <strong> Cell Structure Example </strong>
+  </figcaption>
+</figure>
+
+Receptors: Receive signal from cell surface
+
+Kinase: Reads and decodes the signal
+
+TF: Takes in the signal and triggers production of dna with dna template
+
+Gene: DNA templates
+
+
+We can incorporate such domain knowledge to impose structure on the RVs $$ X_{1},...,X_{8} $$.
+A preliminary way is to partition the RV's into compartments they reside in within a cell.
+Then we can model edges(pathway) that model the dependencies(communication) among the RVs(nodes).
+
+With this structure, we can better express the joint probabilities among the RVs than with a full joint distribution table. The Factorization Law gives us a way to do so. 
+The Factorization Law is a graph traversal algorithm that outputs a unique representation of the joint probability of the RVs. Concisely, we traverse the graph and identify the conditional probabilities of each node given its parent nodes and the marginal probabilities of nodes that do not have parents, then multiply all terms together for the joint probability of all nodes.
+
+There are 3 main benefits of representing the joint distribution in this manner (with a graph structure and conditional probabilities that tie parent nodes and child nodes)
+
+***
+
+## 3 Main Benefits of Graphical Models
+<ol>
+  <li>Cost savings in representing the joint distribution.
+    By modeling the dependencies among the RVs with a graph and conditionals, the number of parameters needed to describe the joint distribution is much fewer than when using a full joint distribution table.
+
+    <dl>Assume all RVs are binary.
+      <dd>$P(X_{1})\rightarrow 1 \text{  parameter}$</dd>  
+      <dd>$P(X_{2})\rightarrow 1 \text{  parameter}$</dd> 
+      <dd>$P(X_{3}|X_{1})\rightarrow 2 \text{  parameters}$</dd> 
+      <dd>$P(X_{4}|X_{2})\rightarrow 2 \text{  parameters}$</dd>
+      <dd>$P(X_{5}|X_{2})\rightarrow 2 \text{  parameters}$</dd>
+      <dd>$P(X_{6}|X_{3}, X_{4})\rightarrow 4 \text{  parameters}$</dd>
+      <dd>$P(X_{7}|X_{6})\rightarrow 2 \text{  parameters}$</dd>  
+      <dd>$P(X_{8}|X_{5}, X_{6})\rightarrow 4 \text{  parameters}$</dd>
+      <dd>Total of 18 parameters</dd>
+    </dl>
+
+    Meanwhile, with a full joint distribution table, we would need $2^{8}-1 $ parameters.
```

Comment:
Could you please switch single `$` to double `$$`? (everywhere in the notes for inline math)
When math is wrapped in single `$`, it doesn't fit well in the line.